sfge/src/main/java/com/salesforce/graph/ops/expander/PathExpansionRegistryUtil.java

```diff
@@ -0,0 +1,65 @@
+package com.salesforce.graph.ops.expander;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Helps translate between lists of instances and their Ids while looking up {@link
+ * PathExpansionRegistry}.
+ */
+public final class PathExpansionRegistryUtil {
+    private PathExpansionRegistryUtil() {}
+
+    /** Convert a list of ApexPathExpander Ids to a list of ApexPathExpander instances. */
+    public static List<ApexPathExpander> convertIdsToApexPathExpanders(
```

Comment:
It feels weird to me that these methods are all defined in a Util instead of on the Registry directly. Seems like most of them could be defined there either as instance methods that use `this` instead of accepting a Registry as a param, or as static methods that accept a list of registries.

Reply:
Sure, let me see how they look in `PathExpansionRegistry`.